client/reader/lib/site-description-blacklist/index.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * External Dependencies
+ */
+import { includes } from 'lodash';
+
+const siteDescriptionBlacklist = [
+	'Just another WordPress.com site',
+	'Just another WordPress site',
+];
+
+/**
+ * Is the provided site description name blacklisted?
```

Comment:
Personally i think its much to make a separate module for this. I can't see it being used outside of `getSiteDescription`.